src/test/scala/edu/ie3/simona/model/participant/load/LoadModelScalingSpec.scala

```diff
@@ -340,7 +340,7 @@ class LoadModelScalingSpec extends UnitSpec with TableDrivenPropertyChecks {
         getRelativeDifference(
           quantile95,
           targetMaximumPower,
-        ) should be < Percent(1d)
+        ) should be < Percent(2d)
```

Comment:
Do we increase this because even the 95 quantile sometimes exceed one percent?

Reply:
Yes. I redruced the sample size from ten years to one year recently, and I tested it 20-30 times, but that wasn't enough apparently.